devdoc/bs_filename_helper.md

```diff
@@ -0,0 +1,30 @@
+`bs_filename_helper` requirements

```

Comment:
>bs_filename_helper [](http://example.com/codeflow?start=1&length=18)

This should likely lose the bs_

Reply:
Same for the other bs_s

Reply:
I can understand how you would want to do that as a second step. Small and steady steps.